documentation/docs/25-build-and-deploy/55-single-page-apps.md

```diff
@@ -9,7 +9,9 @@ You can turn any SvelteKit app, using any adapter, into a fully client-rendered
 export const ssr = false;
 ```
 
-> In most situations this is not recommended: it harms SEO, tends to slow down perceived performance, and makes your app inaccessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
+> SPA mode is highly discouraged: it destroys performance by forcing two network round trips before rendering can begin; harms SEO by downranking your page which is likely to fail core web vitals, excluding search engines that don't render JS, and causing your site to receive less frequent updates from those that do; and, finally, it makes your app inaccessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
+>
+> You can avoid these drawbacks on a given page by [prerendering it](#prerendering-individual-pages). You should thus prerender as many pages as possible when using SPA mode — especially your homepage. If you can prerender all pages, you can simply use [static site generation](adapter-static) rather than a SPA. If you cannot, you should strongly consider using a different adapter.
```

Comment:
The "consider using a different adapter" sentence is a bit confusing, since this page is generally about SPA mode and not about SPA mode in relation to adapter static. Furthermore, it's duplicative to the next paragraph.

Reply:
I just gave another read to the intro section and it's pretty confusing for a couple reasons. One is that it says you can make a SPA with fallback page with any adapter, but I don't think that's true? Most adapters do not have a `fallback` option. It also split the setup instructions with showing how to disable SSR in the intro and the rest of the setup under "Usage". I've tried to restructure it to put all the setup under "Usage" now